platform/mds/mds-web/src/main/java/org/motechproject/mds/web/service/impl/InstanceServiceImpl.java

```diff
@@ -954,6 +962,8 @@ private Object parseValue(ComboboxHolder holder, String methodParameterType, Fie
         } else if (null != holder && holder.isEnumCollection()) {
             String genericType = holder.getEnumName();
             parsedValue = TypeHelper.parse(valueAsString, holder.getTypeClassName(), genericType, classLoader);
+        } else if (parsedValue instanceof UUID) {
```

Comment:
@kmadej I am unable to add a condition in Typehelper.parse, I'm getting checkstyle error (cyclomatic complexity). So, I wrote a new method in TypeHelper to parse string values to uuid   values.